waveorder/util.py

```diff
@@ -13,6 +13,64 @@
 
 numbers = re.compile(r"(\d+)")
 
+import torch
+
+
+def pad_zyx(zyx_data, z_padding):
```

Comment:
Should this be `autp_pad_z` instead?

Reply:
Thanks...poor name. Changed to `pad_zyx_along_z`.